Dockerfile-bats

```diff
@@ -0,0 +1,33 @@
+FROM python:2
+
+ARG SPEC=https://developers.linode.com/openapi.yaml
+ARG TOKEN
+
+RUN apt-get update && apt-get install -y python3 python3-pip bats
+RUN pip install requests terminaltables colorclass PyYAML enum34
+RUN pip3 install requests terminaltables colorclass PyYAML enum34
+
+ENV PYTHONPATH=.
+ENV PATH="/usr/local/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/root/.local/bin"
+
+WORKDIR /src/linode-cli
+
+COPY . .
+
+# Build and Install Linode CLI
+RUN git submodule init \
+    && git submodule update \
+    && make build SPEC=${SPEC} \
+    && make install SPEC=${SPEC} \
+    && cd dist \
+    && pip install --user $(ls) \
+    && echo -n "[DEFAULT]\ntoken = ${TOKEN}" > /root/.linode-cli
```

Comment:
This config file will work, but a real config file would have keys for `image`, `type`, and `region`